Js/localstorage.js

```diff
@@ -0,0 +1,23 @@
+const formData = JSON.parse(localStorage.getItem('form-data')) || {
+  name: '',
+  email: '',
+  message: '',
+};
+
+const inputList = [
+  document.getElementById('name'),
+  document.getElementById('email'),
+  document.getElementById('message'),
+];
+
+function updateFormData() {
+  inputList.forEach((input) => {
+    input.addEventListener('input', () => {
+      formData[input.id] = input.value;
+      localStorage.setItem('form-data', JSON.stringify(formData));
+    });
+    input.value = formData[input.id];
+  });
+}
+
+updateFormData();
```

Comment:
- [ ] On your form, kindly add the picture attached to the section in the bottom right corner as seen on the Figma design 👍 .
- The aim is to have you replicate the design as close as possible, to groom your skills for future circumstances as in a clients project 👍 

## Figma:
![image](https://github.com/alizaamir123/mobile-version/assets/50990925/f9f68363-05ac-44c6-9a66-a9997ea937de)

## Your view:
![image](https://github.com/alizaamir123/mobile-version/assets/50990925/f75b8827-a270-47ee-8f8f-dc22a01a9068)

Reply:
> 

@Shedrack-Sunday  but brother this is not the requirement for this project

Reply:
this projects requirements to only preseve form data in local storage

Reply:
Please as suggested by the previous reviewer, kindly implement the changes requested by adding the required image before asking for a re-review. It will give your project good aesthetics and user interface design 😄